server.go

```diff
@@ -4100,11 +4206,56 @@ func (s *server) OutboundPeerConnected(connReq *connmgr.ConnReq, conn net.Conn)
 		s.removePeer(connectedPeer)
 		s.ignorePeerTermination[connectedPeer] = struct{}{}
 		s.scheduledPeerConnection[pubStr] = func() {
-			s.peerConnected(conn, connReq, false)
+			s.peerConnected(conn, connReq, false, access)
 		}
 	}
 }
 
+// assignPeerPerms assigns a new peer its permissions.
+func (s *server) assignPeerPerms(remotePub *btcec.PublicKey) (peerAccessStatus,
```

Comment:
I find the name `Perms` (permissions) suffix a bit misleading, because every returned status has full permissions ?

Reply:
think it's fine